<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeometryWindowClass</class>
 <widget class="QMainWindow" name="GeometryWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fUIbox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>107</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="swToolBar">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>6</y>
       <width>104</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>243</y>
         <width>114</width>
         <height>101</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pbShowGLview">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>6</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Open GL viewer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>51</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Transparency:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbTransparency">
        <property name="geometry">
         <rect>
          <x>36</x>
          <y>67</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="singleStep">
         <number>0</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pbWebViewer">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>30</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>View geometry using JSROOT</string>
        </property>
        <property name="text">
         <string>JSROOT</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>26</y>
         <width>114</width>
         <height>77</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pbShowGeometry">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>7</y>
          <width>102</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Redraw</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbShowTop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>33</y>
          <width>90</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Show world</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbColor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>48</y>
          <width>101</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Color: material</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>22</y>
          <width>3</width>
          <height>40</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <zorder>line</zorder>
       <zorder>cbShowTop</zorder>
       <zorder>cbColor</zorder>
       <zorder>pbShowGeometry</zorder>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>101</y>
         <width>114</width>
         <height>58</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pbShowTracks">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>30</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Show tracks</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbShowPMnumbers">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>5</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Show PM #s</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>118</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pbHideBar">
       <property name="geometry">
        <rect>
         <x>81</x>
         <y>0</y>
         <width>22</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>157</y>
         <width>114</width>
         <height>59</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pbClearDots">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>31</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear all dots</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pbClearTracks">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>6</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear all tracks</string>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>1</y>
          <width>118</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>214</y>
         <width>114</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pbSaveAs">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>4</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save image</string>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>340</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>214</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>-4</x>
         <y>244</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_8">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>28</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>4</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pbTop">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>345</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Top view</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbFront">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>366</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Front view</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSide">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>387</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Side view</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cobViewType">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>414</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parallel</string>
        </property>
       </item>
      </widget>
      <widget class="Line" name="line_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>441</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>473</y>
         <width>118</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbShowAxes">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>449</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show axes</string>
       </property>
      </widget>
      <zorder>frame_4</zorder>
      <zorder>frame</zorder>
      <zorder>frame_3</zorder>
      <zorder>frame_2</zorder>
      <zorder>frame_5</zorder>
      <zorder>line_6</zorder>
      <zorder>line_5</zorder>
      <zorder>line_7</zorder>
      <zorder>line_8</zorder>
      <zorder>line_2</zorder>
      <zorder>pbHideBar</zorder>
      <zorder>pbTop</zorder>
      <zorder>pbFront</zorder>
      <zorder>pbSide</zorder>
      <zorder>cobViewType</zorder>
      <zorder>line_9</zorder>
      <zorder>line_10</zorder>
      <zorder>cbShowAxes</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="pbShowBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>22</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMarkers">
    <property name="title">
     <string>Markers</string>
    </property>
    <addaction name="actionSmall_dot"/>
    <addaction name="actionLarge_dot"/>
    <addaction name="actionSmall_cross"/>
    <addaction name="actionLarge_cross"/>
    <addaction name="separator"/>
    <addaction name="actionSize_1"/>
    <addaction name="actionSize_2"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionDefault_zoom_1"/>
    <addaction name="actionDefault_zoom_2"/>
    <addaction name="actionDefault_zoom_to_0"/>
   </widget>
   <widget class="QMenu" name="menuLines">
    <property name="title">
     <string>Lines</string>
    </property>
    <addaction name="actionSet_line_width_for_objects"/>
    <addaction name="actionDecrease_line_width"/>
   </widget>
   <addaction name="menuMarkers"/>
   <addaction name="menuLines"/>
   <addaction name="menuZoom"/>
  </widget>
  <action name="actionSmall_dot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixel</string>
   </property>
  </action>
  <action name="actionLarge_dot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large dot</string>
   </property>
  </action>
  <action name="actionSmall_cross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small cross</string>
   </property>
  </action>
  <action name="actionLarge_cross">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large cross</string>
   </property>
  </action>
  <action name="actionSize_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Size +1</string>
   </property>
  </action>
  <action name="actionSize_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Size -1</string>
   </property>
  </action>
  <action name="actionDefault_zoom_1">
   <property name="text">
    <string>Default zoom +1</string>
   </property>
  </action>
  <action name="actionDefault_zoom_2">
   <property name="text">
    <string>Default zoom -1</string>
   </property>
  </action>
  <action name="actionDefault_zoom_to_0">
   <property name="text">
    <string>Default zoom to 0</string>
   </property>
  </action>
  <action name="actionSet_line_width_for_objects">
   <property name="text">
    <string>Increase line width</string>
   </property>
  </action>
  <action name="actionDecrease_line_width">
   <property name="text">
    <string>Decrease line width</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
